harness/tests/cli/test_user.py

```diff
@@ -47,3 +48,23 @@ def test_user_edit_no_fields(mock_die: mock.MagicMock) -> None:
         mock_die.assert_has_calls(
             [mock.call("No field provided. Use 'det user edit -h' for usage.", exit_code=1)]
         )
+
+
+@responses.activate()
+@mock.patch("determined.common.api.authentication.TokenStore", util.MockTokenStore(strict=False))
+@mock.patch("getpass.getpass", lambda *_: "newpass")
+@mock.patch("determined.cli.cli.die")
+def test_login_returns_invalid_credentials_error_message(mock_die: mock.MagicMock) -> None:
```

Comment:
One comment I have left is that the naming on this method is a little wonky since `login` isn't isn't really "returning". "dies with" is the first thing that comes to my own mind, but I'm sure there are other good options, too. Whatever you think is best, no need to check back in.